reference/markdown/Markdown-schema.md

```diff
@@ -0,0 +1,147 @@
+As of version 5.0, the _LwDITA_ plug-in provides the `MarkdownReader` class that supports a new `$schema` key in the YAML front matter block. This key can be used to define parsing options in topic files for more control over how Markdown content is converted to DITA.
+
+```markdown
+---
+$schema: urn:oasis:names:tc:dita:xsd:concept.xsd
+---
+
+# Concept title
+
+Shortdesc content.
+
+Body content.
+```
+
+The `$schema` value is URI that is mapped to a parser configuration. This defined which the document should be parsed, i.e. which Markdown flavour it uses. Markdown schema definition is similar to XML document type declaration or `<?xml-model?>` processing instruction where the document defines how it should be optionally validated.
+
```

Comment:
There should be a paragraph somewhere like this:

> The schema URI may look like it's a reference to an XSD or RNG schema, but it doesn't imply that schema validation will take place.

The current Markdown plug-in just uses those XSD and RNG URIs for convenience because they already exist in the DITA spec. Markdown plug-in doesn't try to search for files references by the schema URI nor does it try to validate against them. Custom parsers may do that and the core Markdown plug-in may do it in the future.

Reply:
Resolved in 364bf47.